app/views/state_file/questions/md_review/edit.html.erb

```diff
@@ -2,5 +2,15 @@
 <% content_for :card do %>
   <%= render "state_file/questions/shared/review_header" %>
 
+  <div id="county-and-subdivision" class="white-group">
+    <div class="spacing-below-5">
+      <p class="text--bold spacing-below-5"><%=t(".county") %></p>
+      <p><%=current_intake.residence_county %></p>
+      <p class="text--bold spacing-below-5"><%=t(".political_subdivision") %></p>
```

Comment:
I think we should use `<strong></strong>` tags instead of the `text--bold` class so that accessibility readers can pick up on the emphasis, unless of course it is purely aesthetic https://accessibility.psu.edu/boldfacehtml/#:~:text=B%20Versus%20STRONG,-Why%20EM%20and&text=STRONG%20and%20EM%20are%20semantic,a%20different%20voice%20or%20style.

Reply:
ohh ty for sharing! love this!